src/index.ts

```diff
@@ -110,6 +110,7 @@ const remarkEmbedder: Plugin<[RemarkEmbedderOptions]> = ({
 
           if (!html) {
             html = await transformer.getHTML(url, config)
+            html = html?.trim() as GottenHTML
```

Comment:
```suggestion
            html = html?.trim() ?? null
```

I think this will be more accurate. Otherwise `html` could be undefined

Reply:
Little teachable moment, thanks!